Foundation/NSXMLParser.swift

```diff
@@ -254,10 +249,8 @@ internal func _NSXMLParserStartElementNs(_ ctx: _CFXMLInterface, localname: Unsa
             var namespaceNameString: String?
             var asAttrNamespaceNameString: String?
             if let ns = namespaces[idx] {
-                if reportNamespaces {
-                    namespaceNameString = UTF8STRING(ns)
-                }
-                asAttrNamespaceNameString = _colonSeparatedStringFromPrefixAndSuffix("xmlns", 5, ns, UInt(strlen(unsafeBitCast(ns, to: UnsafePointer<Int8>.self))))
+                namespaceNameString = UTF8STRING(ns)
```

Comment:
`_colonSeparatedStringFromPrefixAndSuffix` actually unconditionally parses `ns` already as `_colonSeparatedStringFromPrefixAndSuffix` and `UTF8STRING` both call `String._fromCodeUnitSequence`, so if `reportNamespaces` was true it was actually being parsed twice, once into `namespaceNameString` and also into part of `asAttrNamespaceNameString`. This should actually fix it to only be parsed once.